src/single_instruction_test/vm.rs

```diff
@@ -90,7 +87,7 @@ impl<'a> Arbitrary<'a> for VirtualMachine {
                 // Exiting the final frame is different in vm2 on purpose,
                 // so always generate two frames to avoid that.
                 previous_frames: vec![(0, Callframe::dummy())],
-                heaps: u.arbitrary()?,
+                heaps,
```

Comment:
Use the constructor mentioned in previous comment here. That will make the handcrafted fuzzer inputs work again. (They break if the random generation changes even a little bit, which sucks but I think we can get away with it)